java/src/main/java/com/google/cloud/dataproc/templates/util/TemplateConstants.java

```diff
@@ -123,91 +117,71 @@ public interface TemplateConstants {
   String GCS_BQ_INPUT_LOCATION = "gcs.bigquery.input.location";
 
   String GCS_OUTPUT_DATASET_NAME = "gcs.bigquery.output.dataset";
-
   String GCS_BQ_INPUT_FORMAT = "gcs.bigquery.input.format";
-
   String GCS_OUTPUT_TABLE_NAME = "gcs.bigquery.output.table";
-
   String GCS_BQ_CSV_FORMAT = "csv";
-
   String GCS_BQ_AVRO_FORMAT = "avro";
-
   String GCS_BQ_PRQT_FORMAT = "parquet";
-
   String GCS_BQ_CSV_HEADER = "header";
-
   String GCS_BQ_OUTPUT_FORMAT = "com.google.cloud.spark.bigquery";
-
   String GCS_BQ_CSV_INFOR_SCHEMA = "inferSchema";
-
   String GCS_BQ_CSV_DELIMITER_PROP_NAME = "delimiter";
-
   String GCS_BQ_TEMP_BUCKET = "temporaryGcsBucket";
-
   String GCS_BQ_LD_TEMP_BUCKET_NAME = "gcs.bigquery.temp.bucket.name";
-
   String GCS_BQ_TEMP_TABLE = "gcs.bigquery.temp.table";
-
   String GCS_BQ_TEMP_QUERY = "gcs.bigquery.temp.query";
-
   String GCS_BQ_OUTPUT = "table";
-
   String GCS_BQ_AVRO_EXTD_FORMAT = "com.databricks.spark.avro";
 
+  /** GCS to GCS properties */
+  String GCS_GCS_INPUT_LOCATION = "gcs.gcs.input.location";
+
+  String GCS_GCS_INPUT_FORMAT = "gcs.gcs.input.format";
+  String GCS_GCS_CSV_FORMAT = "csv";
+  String GCS_GCS_AVRO_FORMAT = "avro";
+  String GCS_GCS_PRQT_FORMAT = "parquet";
+  String GCS_GCS_CSV_HEADER = "header";
+  String GCS_GCS_CSV_INFOR_SCHEMA = "inferSchema";
+  String GCS_GCS_CSV_DELIMITER_PROP_NAME = "delimiter";
+  String GCS_GCS_AVRO_EXTD_FORMAT = "com.databricks.spark.avro";
```

Comment:
missing changes to template.properties

Reply:
made changes in template properties file as well